fxa/tests/__init__.py

```diff
@@ -1,3 +1,7 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from requests.packages import urllib3
+urllib3.disable_warnings()
```

Comment:
Concretely, my concern is if we accidentally disable a warning about insecure SSL certificate checking or some other security-sensitive matter.  If that's not a possibility then I'm happy to leave it as global disable.  Maybe we can only disable this specific warning rather than all warnings?

Otherwise BaseTestCase seems like a good approach, but I don't want to create too much busywork for future test writers :-(